src/components/GoogleAddressLookup/__test__/component.spec.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import PlacesLookupComponent from './../component';
+
+global.google = {
+    maps: {
+        places: {
+            AutocompleteService: jest.fn(() => ({
+                getPlacePredictions: jest.fn(),
+            })),
+            // PlacesServiceStatus: jest.{
+            //     OK: 'OK',
+            // },
```

Comment:
remove this comments if not used, but if you want to mock PlacesServiceStatus then:
```
PlacesServiceStatus: {
    OK: 'OK',
},
```